addon/components/frost-tabs.js

```diff
@@ -13,7 +13,7 @@ export default Ember.Component.extend(PropTypesMixin, {
 
   propTypes: {
     tabs: PropTypes.array.isRequired,
-    selectedTab: PropTypes.string.isRequired,
+    selectedTab: PropTypes.string,
```

Comment:
What's the state of the component without a tab selected?  Tabs but no content?  (I think it makes sense, just asking)